repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/EventLiteACLEntity.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.hmcts.ccd.definition.store.repository.entity;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Table(name = "event_user_role")
+@Entity
+public class EventLiteACLEntity extends Authorisation implements Serializable {
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "event_id", nullable = false)
+    private EventLiteEntity event;
```

Comment:
Same comment as for case type: Do we need the link to the event entity at all? I feel it would be more straightforward if we only had top-down links for those entities, so event has an ACL, but the ACL doesn't have a link back to the event.

Reply:
removing reverse links to parent entities